### DISCUSSION TITLE
How to pass --repo-update option to cap sync?

### DISCUSSION THREAD

**Discussion by bazuka5801** [ionic-team/capacitor#6548] [Q&A]
How to pass --repo-update option to cap sync?

https://github.com/ionic-team/capacitor/blob/60ffcc36bc4948876b94f1f7a0470c68b6a7e50f/cli/src/ios/update.ts#L113

In this line it not configurable, can we improve this?

**Comment by jcesarmobile** [comment]
What `--repo-update` option does is to run `pod repo update` before `pod install`.
Capacitor doesn't provide a way of passing `--repo-update` to sync command, but you can run `pod repo update` anytime from the ios folder.
